core/camel-management/src/main/java/org/apache/camel/management/mbean/ManagedAsyncProcessorAwaitManager.java

```diff
@@ -82,6 +82,8 @@ public TabularData browse() {
 
     private static CompositeData toCompositeData(AsyncProcessorAwaitManager.AwaitThread entry, CompositeType ct)
             throws OpenDataException {
+        // TODO Once basline is Java 21
```

Comment:
typo basline --> baseline , I guess it wil lbe useful to have the correct workding to easily search for similar todos